### PR TITLE
Update exp_manager for Lightning 1.0

### PR DESCRIPTION
**Pull request by blisc**

### CODE REVIEW

File: nemo/utils/exp_manager.py [hunk 1]
```diff
@@ -519,13 +534,15 @@ def configure_loggers(
 
 
 class NeMoModelCheckpoint(ModelCheckpoint):
+    """ Light wrapper around Lightning's ModelCheckpoint to force a saved checkpoint on train_end
+    """
     @rank_zero_only
     def on_train_end(self, trainer, pl_module):
         filepath = os.path.join(self.dirpath, self.prefix + 'end.ckpt')
         self._save_model(filepath, trainer, pl_module)
```
Comment (okuchaiev) [inline nemo/utils/exp_manager.py:542]: @blisc do you think it make sense to produce ".nemo" file here?

E.g. call ```pl_module.save_to(save_path=os.path.join(self.dirpath, self.prefix + 'end.nemo')```

Comment (blisc) [reply nemo/utils/exp_manager.py:542]: That would make sense. We should do that

**Review by okuchaiev — CHANGES_REQUESTED** [review]
This PR looks good to me.
But could you please modify ASR configs (for speech recognition models only) to monitor: "validation_wer" instead of "val_loss" by default?

**Comment by titu1994** [comment]
@okuchaiev that won't work sadly with multiple data loaders. Each will be tagged with a specific name prepended so exp manager cannot infer it. At least not unless we also attach a "default" validation wer to our logs

**Comment by okuchaiev** [comment]
So can this: "modify ASR configs (for speech recognition models only) to monitor: "validation_wer" instead of "val_loss" by default" be done for a case of single data val loader at least (which is what our configs are)?

**Review by okuchaiev — APPROVED** [review]
discussed offline - asr configs will be fixed in next PR

**Pull request merged**